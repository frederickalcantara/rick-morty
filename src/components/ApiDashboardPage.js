import React from 'react';

export const ApiDashboardPage = () => (
	<div className="box-layout">
	</div>
);