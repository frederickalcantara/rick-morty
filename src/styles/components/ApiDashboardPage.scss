.box-layout {
	background: url('./../img/rick-morty.jpg');
	background-size: cover;
	display: flex;
	justify-content: center;
	align-items: center;
	height: 100vh;
	width: 100vw;
}