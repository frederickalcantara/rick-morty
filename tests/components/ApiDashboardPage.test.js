import React from 'react';
import { shallow } from 'enzyme';
import { ApiDashboardPage } from '../../src/components/ApiDashboardPage';

test('should render ApiDashboardPage correctly', () => {
	const wrapper = shallow(<ApiDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});