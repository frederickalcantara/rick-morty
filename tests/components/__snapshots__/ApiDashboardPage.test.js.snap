// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ApiDashboardPage correctly 1`] = `
<div
  className="box-layout"
/>
`;
